compiler/packages/babel-plugin-react-compiler/src/Babel/RunReactCompilerBabelPlugin.ts

```diff
@@ -36,6 +36,7 @@ export function runBabelPluginReactCompiler(
       "babel-plugin-fbt-runtime",
     ],
     sourceType: "module",
+    configFile: false,
```

Comment:
You may want to also add `babelrc: false` (to disable `.babelrc` in addition to `babel.config`)